plugins/obs-transitions/transition-stinger.c

```diff
@@ -79,6 +79,8 @@ static void stinger_update(void *data, obs_data_t *settings)
 	obs_data_set_bool(media_settings, "is_stinger", true);
 	obs_data_set_bool(media_settings, "is_track_matte", s->track_matte_enabled);
 
+	if (s->media_source && s->transitioning)
+		obs_source_remove_active_child(s->source, s->media_source);
```

Comment:
Prefer an empty line after an unenclosed `if`, or else enclose the if with curly braces.